### PR TITLE
FullStory mobile device mode docs

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
## Description of the change

Implemented FullStory Mobile device mode documentation

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/fullstory.mdx [hunk 1]
```diff
@@ -6,60 +6,237 @@ description: Step-by-step guide to send your event data from RudderStack to Full
...
+  dependencies {
+    classpath 'com.fullstory:gradle-plugin-local:PLUGIN_VERSION'
+    // NOTE: Do not place your application dependencies here; they belong
+    // in the individual module build.gradle files
+  }
+}
+```
+</span>
+</li>
+<li> Make sure that you replace <code class="inline-code">PLUGIN_VERSION</code> with the correct version of the FullStory Android plugin. You can find the latest release notes <a href="https://docs.google.com/document/d/e/2PACX-1vRBDZpgGoLz2IdHntpXtS_1HWGMwlf6n1J1SurkcyguQB9Liv9OCQ1LHnNmfhxwcZ-ZQSl7QUgcMIR5/pub">here</a> (this is currently 1.18.0).
```
Comment (pallabmaiti) [inline destinations/analytics/fullstory.mdx:149]: I believe the latest is 1.21.0

File: destinations/analytics/fullstory.mdx [hunk 2]
```diff
@@ -6,60 +6,237 @@ description: Step-by-step guide to send your event data from RudderStack to Full
...
+<Tabs>
+  <TabList>
+    <Tab>iOS</Tab>
+    <Tab>Android</Tab>
+  </TabList>
+    <TabPanels>
+      <TabPanel>
+        To add FullStory to your iOS project, follow these steps:
+<ul>
+    <li>In your <code class="inline-code">Podfile</code>, add the following extension:
```
Comment (pallabmaiti) [inline destinations/analytics/fullstory.mdx:67]: Replace the extension with dependencies/pod.

File: destinations/analytics/fullstory.mdx [hunk 3]
```diff
@@ -6,60 +6,237 @@ description: Step-by-step guide to send your event data from RudderStack to Full
...
 # FullStory
 
-[FullStory](https://www.fullstory.com/) is a platform for analyzing user interactions, data recording and searching, and much more. It is the perfect platform to measure and improve your overall mobile app experience.
+[**FullStory**](https://www.fullstory.com/) is a platform for analyzing user interactions, data recording and searching, and much more. FullStory's Digital Experience Intelligence (DXI) capability lets you measure and improve your users' overall digital experience.
 
-RudderStack supports sending your event data to FullStory from our native web SDKs, to help you understand your customers better.
+RudderStack supports FullStory as a destination to which you can send your event data seamlessly.
 
-## Getting Started
+## Getting started
```
Comment (pallabmaiti) [inline destinations/analytics/fullstory.mdx:13]: I believe we are using Get started everywhere.

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

File: destinations/analytics/fullstory.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply destinations/analytics/fullstory.mdx:13]: yes - missed this. thanks for pointing out.

File: destinations/analytics/fullstory.mdx (hunk 1 above)
Comment (1abhishekpandey) [reply destinations/analytics/fullstory.mdx:149]: I have did all the testing on FullStory iOS version 1.18.0. 
Should I update that?

File: destinations/analytics/fullstory.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply destinations/analytics/fullstory.mdx:13]: Actually - it's Getting started. While standardizing we will take care of the casing for the rest of the docs.

**Review by pallabmaiti — APPROVED** [review]

**Review by pallabmaiti — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**